### ISSUE TITLE
The interface for obtaining the AF value of IPv6 is not consistent across different platforms.

### ISSUE THREAD

**Issue by zoushucai** [SamuelYvon/netifaces-2#24]
Get the local IPv6

eg:
my win11:

```python
netname = "***"
info = netifaces.ifaddresses(netname)[netifaces.AF_IRDA]  # success

# info = netifaces.ifaddresses(netname)[netifaces.AF_INET6]  # error
```

but  ubuntu 22.04

```python
netname = "***"
info = netifaces.ifaddresses(netname)[netifaces.AF_INET6]   # success
```

**Comment by SamuelYvon** [comment]
What version?

**Comment by zoushucai** [comment]
netifaces2==0.0.21

**Comment by SamuelYvon** [comment]
Re-reading your comment, you are observing that `AF_INET6` (the integer value) differs from platform to platform, correct? IIRC (been a bit) the actual **values** are different but the code does not account for that (ie, `AF_INET6` should always point to ipv6).

This should be simple enough to correct.

**Comment by SamuelYvon** [comment]
Will be fixed in #29
